.github/workflows/test-workflow.yml

```diff
@@ -44,7 +42,7 @@ jobs:
           java-version: ${{ matrix.java }}
       - name: Build and run with Gradle
         working-directory: ${{ env.WORKING_DIR }}
-        run: ./gradlew build ${{ env.BUILD_ARGS }}
+        run: ./gradlew assemble integTest ${{ env.BUILD_ARGS }}
```

Comment:
is this change present in main?

Reply:
I had used the remove argument before, so I will update main with this change as well.